### ISSUE TITLE
Create a link with target

### ISSUE THREAD

**Issue by lilicipute** [adgsm/multi-level-push-menu#83]
Hi,

I am using the basicjs version and I fail to create a simple link.

I suppose I have to replace link: '#' by link: 'mypage.htm' in basicjs.js but it is not working... I can read the link in the statut bar of IE or FF but it does not load.
Moreover, I would like to define a target. Is this option avalaible ?

Thank you in advance, and sorry for spelling errors, I am learning english...

**Comment by FrankHouweling** [comment]
I agree, also thought this was a bug. I would find setting this to false by default a good adjustment.

**Comment by sveliz** [comment]
Hi, @Photospirit  i tried no modify that Option in the file jquery.multilevelpushmenu.js and the jquery.multilevelpushmenu.min.js, and i doesnt work, do u know if i have to change other option.

Thank you.

**Comment by Photospirit** [comment]
Hi sveliz,
I was using it on V2 only. Not sure what version you are using. In the meantime, I'm using a different menu anyway which is similar/same to the one here: http://tympanus.net/codrops/2013/08/13/multi-level-push-menu/

Might be an alternative to try...
Cheers, Chris

**Comment by sveliz** [comment]
Hi @Photospirit , Thanks for the quick reply, i have used the version 2 and i add an option to the next code and it Works Great:

<script>
$(document).ready(function(){
    // HTML markup implementation, overlap mode
    $( '#menu' ).multilevelpushmenu({
        containersToPush: [$( '#pushobj' )],
        preventItemClick: false
    });
});
</script>

See that i add  preventItemClick: false and that's all

Thanks.

**Comment by Coop56** [comment]
I used this as well - setting preventItemClick to false solves the issue and allows you to use a menu item to link to a new tab.
